requires.py

```diff
@@ -206,6 +207,14 @@ def lb_method(self):
         """
         return self._received['lb_method']
 
+    @property
+    def internal_lb(self) -> bool:
+        """
+        If should force internal loadbalancer use.
+        Defaults to false.
+        """
+        return self._received['internal_lb'] or False
```

Comment:
actually, this may not ALWAYS be in the relation data right (ie -- during a charm upgrade)

```suggestion
        return self._received.get('internal_lb',  False)
```

Reply:
Oh good catch :+1: